Makefile

```diff
@@ -231,11 +243,11 @@ ifeq ($(USE_THREADED_ENGINE), 1)
 endif
 
 ifneq ($(ADD_CFLAGS), NONE)
-	CFLAGS += $(ADD_CFLAGS)
+	CFLAGS := $(ADD_CFLAGS) $(CFLAGS)
```

Comment:
Why does the order of the flags need to change?

Reply:
Sorry, I meant to do this only for LDFLAGS, not CFLAGS.